javascripts/discourse/components/topic-in-gated-category.js

```diff
@@ -6,9 +6,12 @@ const enabledCategories = settings.enabled_categories
   .map((id) => parseInt(id, 10))
   .filter((id) => id);
 
+const enabledTags = settings.enabled_tags.split("|");
+
 export default Component.extend({
   tagName: "",
   hidden: true,
+  gatedByTag: false,
```

Comment:
Is there any need for this to be a property of the component? Looks like it's only used inside the `recalculate()` method, so maybe we should just make it a local variable in there?